python/QC3.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+args = parser.parse_args()
+cmd = ["python","$GEM_BASE/python/Produce_Config_File.py"]
+
+for f in args.file:
```

Comment:
more descriptive loop index needed here

Consider using `file` instead of `f`

If `args.file` can be a list of files change to `args.fileList` for example reading this a user may not understand what is being done here